### DISCUSSION TITLE
docs update frequency

### DISCUSSION THREAD

**Discussion by draud3** [pytest-dev/pytest#10982] [General] [answered]
Hello,

I was wondering when the docs available at https://docs.pytest.org/ will be updated after the source of the docs changes: https://github.com/pytest-dev/pytest/tree/main/doc/en

I'm asking this as I noticed the https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html page is not up to date with https://github.com/pytest-dev/pytest/blob/main/doc/en/how-to/tmp_path.rst.

Regards,

**Comment by RonnyPfannschmidt** [comment, marked as answer]
It's in sync with the belonging release Branch, main is the development for the next major /minor release

**Reply by draud3** [reply]
Got it, thanks!
